c/polyjuice.h

```diff
@@ -1020,6 +1020,22 @@ int handle_transfer(gw_context_t* ctx,
   return 0;
 }
 
+int load_eoa_type_hash(gw_context_t* ctx, uint8_t eoa_type_hash[GW_KEY_BYTES]) {
+  mol_seg_t rollup_config_seg;
+  rollup_config_seg.ptr = ctx->rollup_config;
+  rollup_config_seg.size = ctx->rollup_config_size;
+
+  mol_seg_t allowed_eoa_list_seg =
+      MolReader_RollupConfig_get_allowed_eoa_type_hashes(&rollup_config_seg);
+  mol_seg_res_t allowed_type_hash_res =
+      MolReader_AllowedTypeHashVec_get(&allowed_eoa_list_seg, 0);
```

Comment:
allowed eoa: 0: unknown, 1: eth, 2: tron

I think we need to check [AllowedTypeHash.type_](https://github.com/nervosnetwork/godwoken/blob/develop/crates/types/schemas/godwoken.mol#L40)

## How to load eth_eoa_type_hash?
refer to https://github.com/nervosnetwork/godwoken-scripts/blob/master/c/gw_eth_addr_reg.h#L149-L162

Reply:
> [allowed eoa](https://github.com/nervosnetwork/godwoken-scripts/blob/master/c/gw_def.h#L53): 0: unknown, 1: eth, 2: tron
> 
> ## How to load eth_eoa_type_hash?
> I think we need to interate allowed_eoa_type_hashes and find the correct [AllowedTypeHash.type_](https://github.com/nervosnetwork/godwoken/blob/develop/crates/types/schemas/godwoken.mol#L40).
> 
> refer to https://github.com/nervosnetwork/godwoken-scripts/blob/master/c/gw_eth_addr_reg.h#L149-L162

+1

Reply:
Done!